libs/ui/src/lib/static-autocomplete/scope/scope-multiselect.component.html

```diff
@@ -0,0 +1,20 @@
+<mat-form-field appearance="outline">
+  <mat-label>{{label}}</mat-label>
+  <mat-select multiple [formControl]="control" (openedChange)="resetSearch()">
+    <header>
+      <mat-checkbox color="primary" [checked]="checked | async"  [indeterminate]="indeterminate | async" (change)="checkAll($event.checked)"></mat-checkbox>
+      <input [placeholder]="filterPlaceholder" [formControl]="search" />
+      <mat-icon *ngIf="!search.value" svgIcon="search"></mat-icon>
+      <button *ngIf="!!search.value" mat-icon-button type="button" (click)="search.reset('')" matTooltip="Clear text">
```

Comment:
```suggestion
      <button *ngIf="!!search.value" mat-icon-button (click)="search.reset('')" matTooltip="Clear text">
```

And add `MatButtonModule` to module to fix css issue